jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ServiceHandlerTest.java

```diff
@@ -72,7 +72,7 @@ void getServices_withPortsAndWithoutHeadless() {
             .first()
             .satisfies(s -> assertThat(s.getSpec())
                 .hasFieldOrPropertyWithValue("type", "NodePort")
-                .extracting(ServiceSpec::getPorts).asList()
+                .extracting(ServiceSpec::getPorts).asInstanceOf(InstanceOfAssertFactories.list(ServiceSpec.class))
```

Comment:
type inside `list()` should be of list type, in this case it should be `ServicePort`